repose-aggregator/artifacts/valve/src/main/scala/org/openrepose/valve/ReposeJettyServer.scala

```diff
@@ -100,11 +100,20 @@ class ReposeJettyServer(val clusterId: String,
 
       //TODO: do we make this a URL for realsies?
       //Get the configuration root from the core spring context, because we haven't fired up the app Context yet.
+      // See: http://www.eclipse.org/jetty/documentation/current/configuring-ssl.html
       val configRoot = coreSpringProvider.getCoreContext.getEnvironment.getProperty(ReposeSpringProperties.stripSpringValueStupidity(ReposeSpringProperties.CORE.CONFIG_ROOT))
       sslConfig.map { ssl =>
-        cf.setKeyStorePath(configRoot + File.separator + ssl.getKeystoreFilename)
+        cf.setKeyStorePath(new File(configRoot, ssl.getKeystoreFilename).getAbsolutePath)
         cf.setKeyStorePassword(ssl.getKeystorePassword)
         cf.setKeyManagerPassword(ssl.getKeyPassword)
+        val needClientAuth = ssl.isNeedClientAuth
+        cf.setNeedClientAuth(needClientAuth)
+        if (needClientAuth) {
```

Comment:
Since needClientAuth defaults to false, you could simplify this a tiny bit and bring `cf.setNeedClientAuth(needClientAuth)` into this conditional block.

```
if (ssl.isNeedClientAuth) {
  cf.setNeedClientAuth(true)
  ...
}
```